fundamental-concepts/contract-keys/4_using_contract_keys.md

```diff
@@ -2,60 +2,88 @@ All Daml update actions that work on contract IDs also have a counter part for c
 
 | Contract ID | Contract Key  |
 |-------------|---------------|
+| exercise    | exerciseByKey |
 | fetch       | fetchByKey    |
 | lookup      | lookupByKey   |
-| exercise    | exerciseByKey |
 
-Let's try these out in a scenario. First, we let `Alice` exercise the `Follow` choice with
-`exerciseByKey`:
+The above Daml update actions are repreresented by four actions in Daml Script, where `fetch` and `lookup` actions are folded into `queryContract` actions.
+
+<br>
+<table>
+  <tr>
+    <td>Daml</td>
+    <td colspan=2 style="text-align: center;">Daml Script</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>Contract ID</td>
+    <td>Contract Key</td>
+  </tr>
+    <tr>
+    <td>exercise</td>
+    <td>exercsieCmd</td>
+    <td>exerciseByKeyCmd</td>
+  </tr>
+    <tr>
+    <td>fetch/lookup</td>
+    <td>queryContractId</td>
+    <td>queryContractKey</td>
+  </tr>
+</table>
+<br>
+
+Let's try these out with a Daml script. First, we let `Alice` exercise the `Follow` choice with
+`exerciseByKeyCmd`:
 
 <pre class="file" data-filename="daml/User.daml" data-target="append">
-test = scenario do
-  alice <- getParty "Alice"
-  bob <- getParty "Bob"
-  submit alice $ create User with username=alice, following=[]
-  submit alice $ exerciseByKey @User alice $ Follow bob
+test = script do
+  alice <- allocateParty "Alice"
+  bob <- allocateParty "Bob"
+  submit alice $ createCmd User with username=alice, following=[]
+  aliceContractId <- submit alice $ exerciseByKeyCmd @User alice $ Follow bob
+  return()
 </pre>
 
-- `exerciseByKey` takes as a first argument the template name of the contract we want to exercise a
+- `exerciseByKeyCmd` maps to `exerciseCmd` in Daml and takes as a first argument the template name of the contract we want to exercise a
```

Comment:
I don't quite understand this addition. What do you mean by "maps to .. in Daml"?

Reply:
Removed that part, should be cleared now